src/student_success_tool/modeling/utils.py

```diff
@@ -84,59 +76,3 @@ class frequencies in the input data.
         dtype="float32",
         name="sample_weight",
     )
-
```

Comment:
this functionality now found in `student_success_tool.dataio.assets` and `student_success_tool.dataio.models`